js/dom-output.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// On Click, run the .read and .write methods
+document.getElementById("submit").addEventListener("click", function(e) {
+  e.preventDefault();
+
+  var fizzbuzz = new FizzBuzz("butt", "hole");
+  var startVal = Number(document.getElementById("startVal").value);
+  var endVal = Number(document.getElementById("endVal").value);
+  fizzbuzz.read(startVal, endVal);
+  fizzbuzz.write(document.getElementById("output"));
+  console.log(startVal);
```

Comment:
Generally speaking, you want to refrain from leaving `console.log` values in code that gets included in a commit; it's really only useful for debugging and it has potential for impacting performance (imagine a `console.log` statement being run thousands of times as part of a looped function call).